src/components/controllers/repl/index.jsx

```diff
@@ -111,26 +113,17 @@ export class Repl extends Component {
 	}
 
 	componentDidMount() {
-		Promise.all([
-			import('../../code-editor'),
-			import('./runner')
-		]).then(([CodeEditor, Runner]) => {
```

Comment:
No real need to lazily init like this, Vite is smart enough to combine the code editor & runner imports so that tutorial & repl can share them, which is nice.

Might need to keep an eye on that though. Might also revert yet, dunno. Thought I'd try it anyhow.